test/src/test/java/org/corfudb/runtime/collections/SMRMultiLogunitTest.java

```diff
@@ -0,0 +1,216 @@
+package org.corfudb.runtime.collections;
```

Comment:
Yes will do that. These are borrowed tests will get help from the author. Did not want to checkin code without tests.